rero_ils/static/templates/rero_ils/range.html

```diff
@@ -1,9 +1,13 @@
 <div class="panel panel-default" ng-show="vm.invenioSearchResults.aggregations.years.buckets.length > 0">
     <div class="panel-heading">
-        <h3 class="panel-title" translate>years</h3>
+        <h3 class="panel-title">
+          <a data-toggle="collapse" href="#collapse-year" translate>years</a>
+        </h3>
     </div>
-    <div class="panel-body">
-        <div ng-style="{width: '80%', margin: '0 auto'}" id="year_hist"></div>
-        <div ng-style="{width: '80%', margin: '0 auto'}" id="year_select"></div>
+    <div id="collapse-year" class="panel-collapse collapse in">
+      <div class="panel-body">
+          <div ng-style="{width: '80%', margin: '0 auto'}" id="year_hist"></div>
+          <div ng-style="{width: '80%', margin: '0 auto'}" id="year_select"></div>
```

Comment:
Styling into the HTML? Is there another way to do that?

Reply:
Ok will change the search.css.
I don't know who has implemented the styling in the range.html. Do you know a better way, maybe it is better to change it?

Reply:
`git blame` will tell you who did this in the first place. :smiley: 
It should be in a `[sc]ss` somewhere. I have to investigate how and where.

Reply:
@rerowep I tried something for the `range.html`, but it didn't work well. Let's leave it as it is for now.

Reply:
Have updated the CSS and reimplemented the toggle icon. (something went wrong during my last push :-0)